poc/vdaf_pine.py

```diff
@@ -121,6 +135,8 @@ def __init__(self,
             num_wr_checks,
             num_wr_successes,
         )
+        self.flp_squared_norm_equality = FlpBBCGGI19(
+            self.valid_squared_norm_equality)
```

Comment:
Do we actually need to store `valid_squared_norm_equality` in `self`?

Reply:
removed